app/assets/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.js

```diff
@@ -34,7 +34,7 @@
 
   // 2 valid letters - optional space - 6 digits with optional spaces every 2 digits (to account for '12 34 56') - optional space - 1 valid letter
   // e.g. AB123456A or AB 12 34 56 A
-  var NATIONAL_INSURANCE_NUMBER = /[A-CEGHJ-OPR-TW-Z]{2}\s?(\d{2}\s?){3}\s?[A-D]/gi
+  var NATIONAL_INSURANCE_NUMBER = /[A-CEGHJ-OPR-TW-Z]{2}(\s+|\++)?(\d{2}(\s+|\++)?){3}[A-D]/gi
```

Comment:
This better but sorry, should have thought of this sooner - it only matches uppercase letters. We should probably match lowercase as well.

Reply:
Ah, sorry maybe it's not clear enough but with the `/gi`, at the end, the `i` means case insensitive and I've written a test for lowercase too. Should I update the comment to mention it's case insensitive? I probably should add the `+` functionality to the comment anyway.

Reply:
Ah of course it does, sorry hadn't copied and pasted the full regex for testing. Yeah, might be worth updating the comment.

Reply:
No worries, will do - thanks 👍